packages/react-flex/src/components/Flex/Flex.test.tsx

```diff
@@ -12,9 +13,229 @@ describe('Flex', () => {
   /**
    * Note: see more visual regression tests for Flex in /apps/vr-tests.
    */
+
+  it('does not change node structure', () => {
+    const { container } = render(
+      <Flex>
+        <div>Item 1</div>
+        <div>Item 2</div>
+      </Flex>,
+    );
+
+    expect(container).toMatchSnapshot();
+  });
+
   it('renders a default state', () => {
-    const component = renderer.create(<Flex>Default Flex</Flex>);
-    const tree = component.toJSON();
-    expect(tree).toMatchSnapshot();
+    const utils = render(
+      <Flex data-testid="container">
+        <span>item</span>
+      </Flex>,
+    );
+
+    const container = utils.getByTestId('container');
+    expect(container).toHaveStyle({
+      display: 'flex',
+      flexDirection: 'row',
+      justifyContent: 'normal',
+      alignItems: 'normal',
+      flexWrap: 'nowrap',
+    });
+  });
+
+  it('renders default styles for immediate children', () => {
+    const utils = render(
+      <Flex data-testid="container">
+        <span>affected</span>
+        <span>
+          <span>unaffected</span>
+        </span>
+      </Flex>,
+    );
+
+    // TODO: Test initial CSS var values
+
+    const affected = utils.getByText('affected');
+    expect(affected).toHaveStyle({
+      order: 0,
+      margin: 'var(--gap)',
+      flexGrow: 'var(--grow)',
+      flexShrink: 'var(--shrink)',
+      flexBasis: 'auto',
+      alignSelf: 'auto',
+    });
+
+    const unaffected = utils.getByText('unaffected');
+    expect(unaffected).not.toHaveStyle({
+      order: 0,
+      margin: 'var(--gap)',
+      flexGrow: 'var(--grow)',
+      flexShrink: 'var(--shrink)',
+      flexBasis: 'auto',
+      alignSelf: 'auto',
+    });
+  });
+
+  test.each([
```

Comment:
nit: we use `it` across other packages, and this is inconsistent with the previous tests that also use `it`
```suggestion
  it.each([
```